src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -429,14 +429,20 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             }
         }
 
-        private async Task LoadInstalledAndLoadedPackagesAsync() {
+        private async Task LoadInstalledAndLoadedPackagesAsync(bool reload) {
```

Comment:
What happens when user reconnects elsewhere while packages are still being updated? It may be simpler just to close the window. BTW, enumerating packages will put load on the connection and stall REPL intellisense, for example.